common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1267,6 +1267,16 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String WEBCP_LAUNCH_COMPANY_PORTAL_URL = BROWSER_EXT_WEB_CP + "enrollment";
 
+        /**
+         * Redirect URL from WebCP that should launch the enrollment flow.
+         */
+        public static final String WEBCP_ENROLLMENT_URL = "https://enterprise.google.com/android/enroll";
+
+        /**
+         * Redirect URL from WebCP that should launch the authorization flow.
+         */
+        public static final String WEBCP_AUTHORIZE_REDIRECT_URL = "authorize?client_id=74bcdadc-2fdc-4bb3-8459-76d06952a0e9";
```

Comment:
This seems to be just the path and 1 qp vs the full URL. Is this the intention? If yes, let's rename appropriately